src/send/Send.tsx

```diff
@@ -126,22 +128,27 @@ function Send({ route }: Props) {
     // interfere with the invite modal or bottom sheet.
     Keyboard.dismiss()
 
-    if (
-      recipientVerificationStatus === RecipientVerificationStatus.UNVERIFIED ||
-      !hasAddressField(recipient) // should already be caught by UNVERIFIED and UNKNOWN checks, but doing this to be safe and to satisfy the TS compiler
-    ) {
+    if (recipientVerificationStatus === RecipientVerificationStatus.UNVERIFIED) {
       setShowInviteModal(true)
       return
     }
 
     if (getFeatureGate(StatsigFeatureGates.USE_NEW_SEND_FLOW)) {
-      navigate(Screens.SendEnterAmount, {
-        isFromScan: false,
-        defaultTokenIdOverride,
-        forceTokenId,
-        recipient,
-        origin: SendOrigin.AppSendFlow,
-      })
+      if (hasAddressField(recipient)) {
```

Comment:
Does it still handle secure send?

Reply:
no-- we hadn't ported that over to the new flow yet, so this PR doesn't change anything. we're handling it this upcoming sprint